model/container_task_queue_test.go

```diff
@@ -38,8 +39,8 @@ func TestContainerTaskQueue(t *testing.T) {
 		require.False(t, ctq.HasNext())
 		require.Zero(t, ctq.Next())
 	}
-	for tName, tCase := range map[string]func(t *testing.T){
-		"ReturnsAllTasksThatNeedContainerAllocationInOrderOfActivation": func(t *testing.T) {
+	for tName, tCase := range map[string]func(t *testing.T, ctx context.Context){
```

Comment:
nit: it's more conventional in Go for `context.Context` to be the first argument to any function.